test/e2e/setup_test.go

```diff
@@ -36,13 +36,13 @@ const (
 	// The names of the hub cluster + the member clusters set up in this E2E test environment.
 	//
 	// Note that these names must match with those in `setup.sh`, with a prefix `kind-`.
-	hubClusterName     = "kind-hub"
-	memberCluster1Name = "kind-cluster-1"
-	memberCluster2Name = "kind-cluster-2"
-	memberCluster3Name = "kind-cluster-3"
-	memberCluster4Name = "kind-unhealthy-cluster"
-	memberCluster5Name = "kind-left-cluster"
-	memberCluster6Name = "kind-non-existent-cluster"
+	hubClusterName                = "kind-hub"
+	memberCluster1EastProdName    = "kind-cluster-1"
+	memberCluster2EastCanaryName  = "kind-cluster-2"
+	memberCluster3WestProdName    = "kind-cluster-3"
+	memberCluster4UnhealthyName   = "kind-unhealthy-cluster"
+	memberCluster5LeftName        = "kind-left-cluster"
+	memberCluster6NonExistentName = "kind-non-existent-cluster"
```

Comment:
nit: do we need the "Name" as a surfix?

Reply:
Hi Ryan! The suffix is needed as we also have variables named without the suffix in use (`*framework.Cluster`).